public/usage-examples/graphics/fill_rectangle_on_bitmap/fill_rectangle_on_bitmap-1-simple-oop.cs

```diff
@@ -0,0 +1,61 @@
+﻿using SplashKitSDK;
+
+namespace FillRectangleOnBitmap
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            // Create a new window with title and dimensions 
+            SplashKit.OpenWindow("Night Sky", 500, 533);
+            // Create and load new bitmap using the picture file and initialise new bitmap variable
+            Bitmap night_sky = SplashKit.LoadBitmap("night_sky", "sky.jpg"); 
+
+            // Create black rectangles for buildings, with x and y axis and dimensions  
+            SplashKit.FillRectangleOnBitmap(night_sky, Color.Black, 40, 200, 100, 400); // Building 1

```

Comment:
this could be made more OO by calling the fill rectangle from the object such as `night_sky.FillRectangle(Color.Black, 40, 200, 100, 400);` rather than calling through SplashKit namespace.